src/pages/components/progress-bar/usage.mdx

```diff
@@ -95,8 +95,8 @@ which a bar is moving along in constant speed, repeating over time.
 
 | Variant                                                                    | Purpose                                                                                                                                                                                                                                                    |
 | -------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
-| [Determinate](Determinate)                                                 | Use determinate progress bars for a specified amount of time for clear information about the progression — they convey exact and qualitative information about the progression.                                                                            |     |
-| [Indeterminate](/components/progress-bar/usage#indeterminate-progress-bar) | Use indeterminate progress bars for an unspecified completion of time or unclear information about the progression — they convey that users’ request, action, or data is being processed at the moment without indicating how long the activity will take. |
+| [Determinate](Determinate)                                                 | Use determinate progress bars where there is clear information about the progression — they convey exact and qualitative information about the progression.                                                                            |     |
```

Comment:
Isn't "specified" and "unspecified" amount of time an important differentiator  to mention? Why did you think its not necessary, just curious

Reply:
Also I'll defer to @thyhmdo on this change.

Reply:
I really didn't find those terms meaningful. Few author-based progress bars measure a specific _time_, right? It's typically a specific amount of data or percentage of progression. Except in major browser-level downloads, it's rare to see time mentioned (and tends to be woefully inaccurate without quite a bit of data to base that on).

Reply:
I think it makes sense and you're right. Time is usually hard to define and that sentence was too specific about timing while there are other ways to provide this type of information.